lib/libLLVM_julia.jl

```diff
@@ -55,3 +55,7 @@ end
 function LLVMAddJuliaLICMPass(PM)
     ccall(:LLVMExtraJuliaLICMPass,Cvoid,(LLVMPassManagerRef,), PM)
 end
+
+function LLVMAddCPUFeaturesPass(PM)
+    ccall(:LLVMExtraAddCPUFeaturesPass,Cvoid,(LLVMPassManagerRef,), PM)
```

Comment:
not LLVMExtra

Reply:
ahaha, ok then. At least I was consistent, we've apparently named them all like that :slightly_smiling_face: